packages/manager-react-components/src/components/ServiceStateBadge/ServiceStateBadge.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import fr_FR from '@ovh-ux/manager-common-translations/dist/@ovh-ux/manager-common-translations/service/Messages_fr_FR.json';
```

Comment:
this seems risky and also the tests fails because of it, is it the way we should import it ? I don't think so but I'm not sure

Reply:
I think we need build the module before test.